src/shared/cffStandardStrings.js

```diff
@@ -12,6 +12,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* eslint-disable no-var */
```

Comment:
Why disable this here? It looks like `var` can be made `const` in this file without problems?

Reply:
Given that this file isn't actually used for anything, I figured that ignoring it makes sense.
However, I missed that it's already excluded from linting so this line can just be removed; see https://github.com/mozilla/pdf.js/blob/88f72d6b1c37dfc0dbb6715685bf8cb4ae516cd7/.eslintignore#L10-L11